### ISSUE TITLE
blank line needed at start of file?

### ISSUE THREAD

**Issue by AntiSol** [lkrms/pretty-php#60]
Hi there, thanks for pretty-php

I was running it (downloaded .phar version) over a project and ran into the following when it was trying to process a particular file:
```
Call to a member function declarationParts() on bool in phar:///usr/local/bin/pretty-php/src/Rule/DeclarationSpacing.php:218
```

Unfortunately I can't show you the code, but I can give you a close approximation of the first few lines of the file (no blank lines, no comments immediately after opening php tag):
```
<?php
date_default_timezone_set("Relevant/Timezone");
global $bad_practice;
define('SOME_VAR', 42);
/**
 * comment
 */
class SomeClass {
 ...
```

I did some messing around and found that I was able to work around this by inserting some blank lines at the start of the file, thusly:
```
<?php


date_default_timezone_set("Relevant/Timezone");
...
```

Hope this helps someone! :)

**Comment by lkrms** [comment]
Thanks for the detailed bug report :)

I've been able to replicate the issue here and will hopefully have it fixed shortly.

**Comment by AntiSol** [comment]
Awesome! I'm glad I could help! :grin:

**Comment by lkrms** [comment]
Fix released in v0.4.36. Please let me know if you have any other issues! Thanks again :)
